contact.ipynb

```diff
@@ -0,0 +1,158 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "a1b1dec1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "220b7b84",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<_io.TextIOWrapper name='contact.csv' mode='w' encoding='cp1252'>\n"
+     ]
+    }
+   ],
+   "source": [
+    "f=open('contact.csv','w',newline='')\n",
```

Comment:
if there any other way to open a file.

Reply:
Using a with statement: 
with open('contact.csv', 'w', newline='') as f: